uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/apis/entitlement-api-mock.js

```diff
@@ -0,0 +1,23 @@
+//intercepts paths for real APIs and returns mock data
+define([
+  'mocks/module',
+  'mocks/apis/data/entitlement-data',
```

Comment:
I'm commenting this line just to comment on the folder path of the mocked data. It should be `test/mocks/data`, not `test/mocks/apis/data`